src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -25,7 +25,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <ContainerName Condition="'$(ContainerName)' == ''">$(TestProduct)-$(Branch)-$(BuildMoniker)$(CloudTestContainerSuffix)</ContainerName>
+    <ContainerName>build-$([System.Guid]::NewGuid().ToString("N"))</ContainerName>
```

Comment:
We should be able to override a "ContainerName".  Please add back the Condition.